### DISCUSSION TITLE
Lisa Source code released

### DISCUSSION THREAD

**Discussion by MarkAlanJones** [robhagemans/hoard-of-bitfonts#11] [General]
https://info.computerhistory.org/apple-lisa-code

Fonts are in binary .F format

https://www.zdnet.com/article/this-1980s-computer-was-a-huge-leap-forward-now-you-can-download-its-source-code/

**Comment by robhagemans** [comment]
Thanks for the link!

I had a look at `HELV18C.F`, it has an offset table and a width table at the end and a 240-bytes wide bitmap strike.
(Presumably the source code describes the format somewhere too)

![HELV18C](https://user-images.githubusercontent.com/7723154/214227191-5e48e188-ac0a-4a5e-92f3-be1522ea240c.png)

**Reply by robhagemans** [reply]
Ah, they're actually just `NFNT` resources.

**Comment by robhagemans** [comment]
Oddly the system font itself seems not to be included - from what other sources say this would be expected in a file named `SYSTEXT` or `SYSTEXTS.F`. On the compiled system all of the fonts would be in `font.lib` which presumably would be compiled from the `.F` resources given here. I wonder if the file was left out or if the font is defined elsewhere in these sources.

**Comment by MarkAlanJones** [comment]
I'm just hoping there was some unique Lisa fonts that didn't  just get copied to the Mac 😸
